test/test_utils/__init__.py

```diff
@@ -1447,6 +1507,18 @@ def get_unique_name_from_tag(image_uri):
     return re.sub("[^A-Za-z0-9]+", "", image_uri)
 
 
+def get_image_type_from_tag(image_uri):
+    valid_image_types = ["training", "inference"]
+    image_type_match = [keyword for keyword in valid_image_types if keyword in image_uri]
+    if len(image_type_match) != 1:
+        raise LookupError(f"Failed to find whether {image_uri} is training or inference")
+    return image_type_match[0]
+
+
+def get_image_arch_type_from_tag(image_uri):
+    return "graviton" if "graviton" in image_uri else "x86"
```

Comment:
This is probably sufficient for now, but it may be worth noting in the function description that this relies on the assumption that everything that is not graviton is x86.